reactor-netty-examples/src/main/java/reactor/netty/examples/http/snoop/HttpSnoopClient.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2023 VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.examples.http.snoop;
+
+import io.netty.handler.codec.http.HttpHeaderNames;
+import io.netty.handler.codec.http.HttpHeaderValues;
+import io.netty.handler.ssl.util.InsecureTrustManagerFactory;
+import reactor.core.publisher.Mono;
+import reactor.netty.ByteBufFlux;
```

Comment:
```suggestion
import reactor.netty.ByteBufMono;
```

@longkunbetter ,

thanks for the PR. 

My two cents: please replace the old import of `reactor.netty.ByteBufFlux` to `reactor.netty.ByteBufMono`, because the example is not currently compiling.